test/integration/targets/connection_winrm/tests.yml

```diff
@@ -41,3 +41,20 @@
   - assert:
       that:
       - timeout_cmd.msg == 'The win_shell action failed to execute in the expected time frame (5) and was terminated'
+
+  - name: get WinRM quota value
+    win_shell: (Get-Item WSMan:\localhost\Service\MaxConcurrentOperationsPerUser).Value
+    changed_when: false
+    register: winrm_quota
+
+  - block:
+    - name: set WinRM quota to lower value
```

Comment:
Minor nit- this test change should also include marking `connection_winrm` with the `destructive` alias. If someone Ctrl-C's the test run in between, the quota value on the target will likely be permanently lowerered.

Reply:
Good point, I actually did that at one point and had to manually reset it :)